packages/front-end/components/SyntaxHighlighting/Snippets/GrowthBookSetupCodeSnippet.tsx

```diff
@@ -441,13 +441,26 @@ TrackingCallback trackingCallback = new TrackingCallback() {
         Create a GrowthBook instance
         <Code
           language="java"
-          code={`
+          code={
+            encryptionKey
+              ? `
 GBContext context = GBContext.builder()
     .featuresJson(featuresJson)
+    .attributesJson(userAttributesObj.toString()) // Optional
```

Comment:
Also adding this here since it can be implemented as either part of the builder or using the `setAttributes` method.